client/router_create.go

```diff
@@ -1079,7 +1079,19 @@ sasldb_path: /tmp/skrouterd.sasldb
 			}
 		}
 	}
-	dep, err := kube.NewTransportDeployment(van, siteOwnerRef, cli.KubeClient)
+
+	runAsNonRoot := true
+	podSecurityContext := corev1.PodSecurityContext{
```

Comment:
One comment/question here. The `corev1.PodSecurityContext` type is not
the same one we are setting at the YAML files, which uses `v1.SecurityContext` type,
and that one is defined at the container level.

Should the same fields be set on both places?

Reply:
Agree and will update